packages/untp-test-suite/src/interfaces/utils/common.ts

```diff
@@ -9,4 +18,44 @@ export async function readJsonFile<T>(filePath: string): Promise<T | null> {
   } catch (error) {
     return null;
   }
-}
\ No newline at end of file
+}
+
+export const getTemplateName = (testSuiteResult: IValidatedCredentials): string => {
```

Comment:
I think `getTemplateName` and `generateFinalMessage` should be in the `templates` folder, these functions do not deal with the interface (CLI or Lib)